python/dgl/utils.py

```diff
@@ -126,9 +127,61 @@ def __getstate__(self):
     def __setstate__(self, state):
         self._initialize_data(state)
 
+    def get_items(self, index):
```

Comment:
We need docstring for `get_items` and `set_items`. What is the meaning of `index`? I found you have three scenarios (1) advanced index, (2) boolean mask and (3) slice. If so, please make sure that the correct backend function is called (e.g. `gather_rows` and `narrow`). Boolean mask is currently missing in the backend. You could add it.

Reply:
I don't think we have case (2).

There are two possible cases: advanced index and slice. I feel like tensors from most (all) backends support slicing. And for slice type, the step may not be zero, so I cannot use narrow.